apps/beeai-cli/src/beeai_cli/commands/resource_server.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2025 © BeeAI a Series of LF Projects, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+from beeai_cli.async_typer import AsyncTyper, console
+from beeai_cli.configuration import Configuration
+
+app = AsyncTyper()
+
+config = Configuration()
+
+
+@app.command("show")
+def show_server():
+    active = config.auth_manager.get_active_resource()
+    if not active:
+        console.print("No active server!!!\n")
```

Comment:
Use `console.error`, `console.success` etc. for messages, see other modules for examples. (Applies to most `console.print` calls here.)